feature-libs/product-configurator/rulebased/components/attribute/types/base/configurator-attribute-single-selection-base.component.ts

```diff
@@ -155,4 +167,26 @@ export abstract class ConfiguratorAttributeSingleSelectionBaseComponent extends
   protected getSelectedValuePrice(): Configurator.PriceDetails | undefined {
     return this.attribute.values?.find((value) => value.selected)?.valuePrice;
   }
+
+  protected get isAdditionalValue(): boolean {
+    return (
+      this.attribute.uiType ===
+        Configurator.UiType.RADIOBUTTON_ADDITIONAL_INPUT ||
+      this.attribute.uiType === Configurator.UiType.DROPDOWN_ADDITIONAL_INPUT
+    );
+  }
```

Comment:
the same method exists in the attribute-header-component just named slighty different 'isWithAdditionalValues'.  Maybe this could be refactred into the 'ConfiguratorAttributeBaseComponent' so the code exist only onnce.

Reply:
Done